src/LineUI/LineSet.tsx

```diff
@@ -203,12 +211,27 @@ const LineSet : React.FC<ILineSetProps> = ({ getCTM, boundaries, unit, size, lin
     />
   );});
 
+  const AreaLabel = () => {
+    const pointsLength = lineSetData.points.length;
+    if (pointsLength < 3) return null;
+    let midpoint = { x: 0, y: 0 };
```

Comment:
const

Reply:
We are reassigning value to midpoint

Reply:
meh ...IMO this means you could use a reduce?